quiz/generic/wizard/default_messages.php

```diff
@@ -4,12 +4,9 @@
 include_once($relPath.'theme.inc');
 include_once('../quiz_defaults.inc'); // $messages
 
-$theme_args["css_data"] = "h2 {font-size:110%; margin: 0;}
-    p.message_id {margin: 0 0 .5em 1em; background-color: #dddddd;}";
```

Comment:
Let's remove the reference to this on line 18 as well.

Reply:
Yes, let's! And thanks for pointing that out. I totally forgot to look for it.

Done!